backend/tracim_backend/applications/upload_permissions/email_manager.py

```diff
@@ -0,0 +1,242 @@
+from email.message import Message
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from email.utils import formataddr
+import typing
+
+from tracim_backend.applications.upload_permissions.models_in_context import (
+    UploadPermissionInContext,
+)
+from tracim_backend.lib.core.workspace import WorkspaceApi
+from tracim_backend.lib.mail_notifier.notifier import EmailManager
+from tracim_backend.lib.mail_notifier.sender import EmailSender
+from tracim_backend.lib.mail_notifier.sender import send_email_through
+from tracim_backend.lib.utils.logger import logger
+from tracim_backend.lib.utils.translation import Translator
+from tracim_backend.models.auth import User
+from tracim_backend.models.context_models import ContentInContext
+from tracim_backend.models.data import Workspace
+
+
+class UploadPermissionEmailManager(EmailManager):
+    def notify_new_upload(
+        self,
+        uploader_username,
+        workspace: Workspace,
+        uploaded_contents: typing.List[ContentInContext],
+    ) -> None:
+        email_sender = EmailSender(
+            self.config, self._smtp_config, self.config.EMAIL__NOTIFICATION__ACTIVATED
+        )
+        notifiable_roles = WorkspaceApi(
+            current_user=None, session=self.session, config=self.config
+        ).get_notifiable_roles(workspace, force_notify=True)
```

Comment:
Force notification is in spec ?

Reply:
Spec does say we should notify all "active" user in workspace, this mean all workspace users with active account, users with notifications disabled on workspace included. That's the meaning of force_notify here.

Reply:
@lebouquetin @PhilippeAccorsi This question don't block this MR; Is that a good idea to notify all workspace active (even if) users when upload ?

Reply:
I think its a good idea for many reasons: 
- user that give access is deactivated when file is uploaded
- one other user wait for this file uploaded
- if all users not receive notification, you can't know that new files are uploaded
Maybe more reason.

Reply:
Okay if it is obviousness for you. But i still think it is weird to notify persons who disable notifications.

Reply:
I know this is a conclusion of @lebouquetin about notify all user also user have deactivated notification.
This is a collaborative tool so If it's impossible to inform user in some special case, you can't work whis all members, or you need to use another tool to inform this kind of users.